test/test_utils/__init__.py

```diff
@@ -1065,7 +1071,7 @@ def get_framework_and_version_from_tag(image_uri):
             "2.2.3": "2.2.3.2.0.3.0",
             "2.3.4": "2.3.4.2.0.3.0",
             "2.4.3": "2.4.3.2.0.3.0",
-            "2.5.1": "2.5.1.2.0.3.0",
+            "2.5.1": "2.5.0.2.0.3.0",
```

Comment:
Why was this change reverted?

Reply:
oh ok..this was a mistake while doing the merge from the latest master. Will fix it. Thanks